shared/datadog/datadog_test.go

```diff
@@ -0,0 +1,35 @@
+package datadog
+
+import (
+	"net"
+	"os"
+	"testing"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/stretchr/testify/assert"
+)
+
+var addr, _ = net.ResolveUDPAddr("udp", "localhost:8125")
+var sock, _ = net.ListenUDP("udp", addr)
+
+func TestLogInfo(t *testing.T) {
+	defer func() {
+		buf := make([]byte, 1024)
```

Comment:
Why run these from a defer and not directly at the end of the test?

Reply:
My bad.  Moved it to end of test.

Reply:
I don't think you did?